### PR TITLE
Feature/cc 1 config database connection

### PR DESCRIPTION
**Pull request by Lexander1108**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,14 +15,24 @@ app.use(
...
-  const cases = await casesService.getCases();
+  const cases = await casesService.getCases(request.query);
   response.status(200).json(cases);
 });
 
 app.listen(process.env.PORT, () => {
 	console.log(`Server started on port ${process.env.PORT}.`);
 });
+
+let newCase =
```
Comment (DrSmile444) [inline src/index.js:30]: You don't need this code in stage. If you need it for debugging or testing, remove it and don't push

File: src/services/index.js [hunk 2]
```diff
@@ -6,3 +6,5 @@ export const casesService = new CasesService(postgresService);
 
 export * from './cases.service.js';
 export * from './postgres.service.js';
+
+// casesService.getCases().then(console.log);
```
Comment (DrSmile444) [inline src/services/index.js:10]: Remove this please

File: src/services/postgres.service.js [hunk 3]
```diff
@@ -1,17 +1,20 @@
 import pg from 'pg';
 
+import knex from 'knex';
+
 export class PostgresService {
   constructor() {
     this.pool = new pg.Pool();
```
Comment (DrSmile444) [inline src/services/postgres.service.js:7]: Do you still need the pool? If not remove this line and pg import

File: src/index.js (hunk 1 above)
Comment (Lexander1108) [reply src/index.js]: Okay, got it

File: src/services/postgres.service.js (hunk 3 above)
Comment (Lexander1108) [reply src/services/postgres.service.js]: My bad. Pool deleted

File: src/services/index.js (hunk 2 above)
Comment (Lexander1108) [reply src/services/index.js]: Okay. Thank you

**Pull request merged**